includes/incoming-events/class-user-updated.php

```diff
@@ -76,6 +76,24 @@ public function maybe_update_user() {
 			}
 		}
 
+		// @TODO move this to the User Utils class when it gets merged.
```

Comment:
Merged in #8

Reply:
yeah... but then I realized this is only going to be used in another place after https://github.com/Automattic/newspack-network/pull/17 is merged.

Because there, we are creating the authors but we are not syncing the avatar the first time the author is distributed. So when that gets merged, then we need to make avatars also synced when the author is first created... and then we can move this